client/BidBuy.jsx

```diff
@@ -77,7 +77,8 @@ class BidBuy extends React.Component {
 
   fetchItemById(id) {
     fetch(`${hostURL}/items/id/${id}`)
-      .then(response => response.json())
+      .then(response => response.text())
+      .then(res => console.log(res))
```

Comment:
Is it better to use .tap here since it's just a console log?

Reply:
`Tap` would be nice here, but I am not using `bluebird`!